packages/browser/src/browser/__tests__/inspector.integration.test.ts

```diff
@@ -1,5 +1,6 @@
+import { waitFor } from '@testing-library/dom'
```

Comment:
I love this lib, but I feel like this helper (while useful) is a bit misleading here, as there's no relevant DOM manipulation going on in these tests.

Reply:
I mentioned the alternative https://github.com/segmentio/analytics-next/pull/764#discussion_r1066397210

Reply:
It won't work. The returned promise can't be `await`ed because it never resolves, or throws.

Reply:
> It won't work. The returned promise can't be awaited because it never resolves, or throws.

Did you try running the test? It _does pass_, I just checked.

Reply:
https://github.com/segmentio/analytics-next/pull/764#discussion_r1070112271